articles/migrations/guides/liveid-api-deprecation.md

```diff
@@ -0,0 +1,135 @@
+---
+title: Migration to Azure AD Personal Accounts
+description: This article covers the Live Connect + SDK deprecation and how to update your Auth0 Microsoft Account Connection.
+toc: true
+contentType:
+  - how-to
+useCase:
+  - add-login
+  - migrate
+---
+
+# Microsoft Account Migration to Azure AD (personal accounts) + Microsoft Graph
+
+In October 2017, Microsoft announced the [deprecation of the Live Connect API and Live SDK](https://developer.microsoft.com/en-us/office/blogs/outlook-rest-api-v1-0-office-365-discovery-and-live-connect-api-deprecation). This is a Microsoft deprecation that will affect Auth0 users using the Microsoft social connection, and requires changes in the Auth0 configuration and potentially application code. 
+
+The change implies switching:
```

Comment:
I would say something like "The change implies switching how Auth0 interacts with the Microsoft authentication APIs", to make it clearer that the following changes don't need to happen in the customer's applications.